design-system/react/src/components/Stack/index.tsx

```diff
@@ -1 +1,2 @@
-export * from './Stack';
+export * from '../Box';
+export { Stack } from '../Box/Stack';
```

Comment:
I think we should break the compatibility here and force users to use `Box.Stack`

Reply:
✅